helm/private/helm_package.bzl

```
@@ -162,11 +162,15 @@ def _helm_package_impl(ctx):
 
     args.add("-workspace_name", ctx.workspace_name)
 
+    for ln in ctx.attr.files:
```

Kinda hard to use suggestions to suggest exactly what I'm thinking but:
```suggestion
    def _rlocationpath(file, workspace_name):
        if file.short_path.startswith("../"):
            return file.short_path[len("../"):]

        return "{}/{}".format(workspace_name, file.short_path)

    workspace_name = ctx.workspace_name

    def _rlocationpath_mapping(file):
        return "-add_files={}={}".format(file.path, _rlocationpath(file, workspace_name))

    args.add_all(ctx.files.files, map_each = _rlocationpath_mapping, allow_closure = True)
```

Anyone familiar with Runfiles APIs (e.g. [rust](https://github.com/bazelbuild/rules_rust/blob/95563e59098864e566395b6f0dbc7068f03ed0a8/tools/runfiles/runfiles.rs#L1-L31), [python](https://github.com/bazelbuild/rules_python/tree/0.36.0/python/runfiles)) should have an easy time finding files and not need to refer to the BUILD file target to understand what files are included and where to find them.